cartridge/argparse.lua

```diff
@@ -279,7 +283,13 @@ local function load_dir(dirname)
                     section_name, dirname, origin[section_name], fio.basename(f)
                 )
             end
-
+            if type(content) ~= 'table' then
+                return nil, ParseConfigError:new(
+                    'invalid content found in file %s in section %s: '..
+                    'expected a table of key-value pairs, got %s',
+                    fio.basename(f), section_name, type(content)
+                )
+            end
```

Comment:
I believe this error is not necessary here. As I understand this code, we are loading directory into a flat config meaning each section has its own config.

But in some cases we parse sections, which are not needed actually. We can only figure it out later in the `parse_file` function, when we filter out all unnecessary sections.

But what to do here, when we do not have all the information which sections we needed? I suggest to just ignore invalid sections here, because what is a section?

Section is something with a name and a key-value map:

```yaml
section_name:
  key1: value1
  key2: value2
  # ...
```

But something like `section_name: a_string` is not actionally a section by definition. Why not just ignore it here?

Reply:
Ok, let's ignore everything.